e2e-tests/manifests/rapidast-oobtkube-configmap.yaml

```diff
@@ -20,7 +20,9 @@ data:
         #   if it is "*stdout" or unspecified, the command's standard output will be selected
         results: "/tmp/oobtkube.sarif.json"   # if None or "*stdout", the command's standard output is selected
         # toolDir: scanners/generic/tools
-        inline: "python3 oobtkube.py --log-level debug -d 60 -p 6000 -i rapidast-oobtkube -f /opt/rapidast/config/cr_example.yaml | tee /tmp/oobtkube.sarif.json"
+        inline: |
+          PYTHON_VERSION="${PYTHON_VERSION:-python3}"
```

Comment:
Not for this PR, as an aside, should we bump to python3.12 for vanilla rapidast as well, so that both images use the same version?